integration_test/Makefile

```diff
@@ -148,3 +148,21 @@ pg12:
 	fi
 	jq '.queryInformations[].normalizedQuery' pg12_2.json | sort --version-sort -f > pg12.out
 	diff -Naur pg12.expected pg12.out && echo 'success'
+
+pg13:
+	docker build -f Dockerfile.test-pg13 $(DOCKER_BUILD_OPTS)
+	$(DOCKER_RUN_CMD)
+	sleep 10
+	docker exec pganalyze-collector-test pgbench -U postgres -i
+	docker exec pganalyze-collector-test pgbench -U postgres
+	docker exec pganalyze-collector-test sh -c "DB_USERNAME=postgres PGA_API_KEY=dummy PGA_DISABLE_ACTIVITY=1 pganalyze-collector --dry-run --force-state-update -v" > pg13_1.json
+	docker exec pganalyze-collector-test pgbench -U postgres
+	docker exec pganalyze-collector-test sh -c "DB_USERNAME=postgres PGA_API_KEY=dummy PGA_DISABLE_ACTIVITY=1 pganalyze-collector --dry-run --force-state-update -v" > pg13_2.json
```

Comment:
Why do we run the test twice?

Reply:
In order to get the query statistics - the first run only initializes the state file, and doesn't send any statistics.

Technically we could be doing `> /dev/null` for the first run, but if there are errors it's helpful to have that output around.